docs/en/upgrade.md

```diff
@@ -63,11 +63,18 @@ The solution automatically and asynchronously upgrades the views and materialize
 
 ### Migrate the Existing Data (only applicable when upgrading from a version earlier than v1.1.6)
 
-!!! info "Important"
+!!! danger "Important"
 
     The data migration process is CPU-intensive. Before starting the migration, ensure that the load on your Redshift is low. It's also advisable to consider temporarily increasing the RPUs of Redshift Serverless or the cluster size when migrating large volumes of data.
```

Comment:
--> The data migration process is CPU-intensive, and it will **incur additional cost**.

Reply:
added.